pkg/kubernetes/pod/v1alpha1/pod.go

```diff
@@ -76,6 +76,32 @@ func IsRunning() Predicate {
 	}
 }
 
+// HasLabels returns true if provided labels
+// map[key]value are present in the provided PodList
+// instance
+func HasLabels(keyValuePair map[string]string) Predicate {
+	return func(p *Pod) bool {
+		//		objKeyValues := p.object.GetLabels()
+		for key, value := range keyValuePair {
+			if !p.HasLabel(key, value) {
+				return false
+			}
+		}
+		return true
+	}
+}
+
+// HasLabel return true if provided lable
+// key and value are present in the the provided PodList
+// instance
+func (p *Pod) HasLabel(key, value string) bool {
```

Comment:
Expose a predicate as well.
Let this predicate use this method

Reply:
Added another method